Add Azure DevOps Plugin (#241)
janus-idp/backstage-showcase: showcase-docs/getting-started.md

       apiKey: ${JENKINS_TOKEN}
   ```

   - Enable plugins (All plugins have a default of `false`)

     - `${K8S_ENABLED}`: Set to `true` to enable the Kubernetes backend plugin.
     - `${TECHDOCS_ENABLED}` Set to `true` to enable the Techdocs backend plugin.
     - `${ARGOCD_ENABLED}` Set to `true` to enable the ArgoCD backend plugin.
     - `${SONARQUBE_ENABLED}` Set to `true` to enable the SonarQube backend plugin.
     - `${KEYCLOAK_ENABLED}` Set to `true` to enable the Keycloak backend plugin.
     - `${OCM_ENABLED}` Set to `true` to enable the OCM backend plugin.
     - `${GITHUB_ENABLED}` Set to `true` to enable the GitHub Entity backend plugin.
     - `${GITHUB_ORG_ENABLED}` Set to `true` to enable the GitHub Org Entity backend plugin.
     - `${GITLAB_ENABLED}` Set to `true` to enable the GitLab Entity backend plugin.
     - `${AZURE_ENABLED}` Set to `true` to enable the Azure DevOps Entity backend plugin.

>      - ${JENKINS_ENABLED}`Set to`true` to enable the Jenkins Entity backend plugin.

Reviewer: ```suggestion
     - `${JENKINS_ENABLED}` Set to `true` to enable the Jenkins Entity backend plugin.
```

   - Setup the GitHub plugins (GitHub Issues and GitHub Pull Request)

     - This [URL](https://backstage.io/docs/integrations/github/github-apps) can be used to quickly create a GitHub app, you can name the yaml file `github-app-backstage-showcase-credentials.local.yaml`
     - `${GITHUB_APP_CLIENT_ID}`: client id
     - `${GITHUB_APP_CLIENT_SECRET}`: client secret

   - Setup the GitLab plugin

     - This [URL](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) can be used to quickly create a GitLab personal access token
     - `${GITLAB_TOKEN}`: personal access token

   - Setup the Azure DevOps plugin

>      - This [URL](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows) can be used to quickly create a GitLab personal access token

Reviewer: ```suggestion
     - This [URL](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows) can be used to quickly create an Azure personal access token
```
